### DISCUSSION TITLE
Detox unable to tap listview item

### DISCUSSION THREAD

**Discussion by Zigiza** [wix/Detox#4502] [Q&A]
Hi, 

I am having trouble tapping on a listview item on an iOS app on an apple simulator using detox.

I have tried using an ID and label to identify the item but the script hangs when it comes to tapping on the object.
 I am looking for tips or some guide to tapping on listview elements?

Thank you

**Comment by asafkorem** [comment]
Shouldn't be anything special with listview items. Can you provide some lean example of a listview that you're trying to interact with?

**Comment by asafkorem** [comment]
@Zigiza can you elaborate what happens while it hangs? Do you get any error messages or synchronization logs? (app is busy, etc.)
